lib/utilization/docker-info.js

```diff
@@ -197,7 +197,7 @@ function fetchDockerVendorInfo(agent, callback, logger = log) {
  * @param {object} [logger] internal logger instance
  */
 function parseCGroupsV2(data, callback, logger = log) {
-  const containerLine = new RegExp('/docker/containers/([0-9a-f]{64})/')
+  const containerLine = new RegExp('/containers/([0-9a-f]{64})/')
```

Comment:
I get what this PR is doing but this is going to impact every language agent. I have put this on the agent arch working group agenda to discuss because we don't even have tests that only have container ids in `/containers/<id>` vs `/docker/containers/<id>`

Reply:
I've added [a cross-agent test for this](https://github.com/newrelic/node-newrelic/pull/2419). The infra team's identified that the `mountinfo` on Ubuntu 20.04.6 expresses its cgroup v2 container ID differently from the other versions we've tested. 

Current Slack discussion here: https://newrelic.slack.com/archives/C0511B24H/p1722379443942719